web/src/components/Presentation/AttributeList.vue

```diff
@@ -128,11 +136,13 @@ export default defineComponent({
         v-for="({ name, target }) in alternateIdentifiers"
         :key="name"
         v-bind="{ item, field, link: { name, target } }"
+        :image="name.startsWith('gold') ? images.gold : name.startsWith('img') ? images.img : undefined"
```

Comment:
Minor issue, but the prop definition in [AttributeItem.vue ](https://github.com/microbiomedata/nmdc-server/blob/da8bb307b45492fe971f05c62d5d0d5ee7b60957/web/src/components/Presentation/AttributeItem.vue#L31-L33) has it typed as a string, so it might be a little cleaner to pass in `''` if no image matches.

```suggestion
        :image="name.startsWith('gold') ? images.gold : name.startsWith('img') ? images.img : ''"
```

Reply:
Thanks!